src/store/conversationsStore.js

```diff
@@ -240,13 +240,21 @@ const actions = {
 	 */
 	postAddConversation(context, conversation) {
 		if (conversation.type === CONVERSATION.TYPE.ONE_TO_ONE && conversation.status) {
-			emit('user_status:status.updated', {
-				status: conversation.status,
-				message: conversation.statusMessage,
-				icon: conversation.statusIcon,
-				clearAt: conversation.statusClearAt,
-				userId: conversation.name,
-			})
+			const prevStatus = context.state.conversations[conversation.token] || {}
+
+			// Only emit the event when something actually changed
+			if (prevStatus?.status !== conversation.status
+				|| prevStatus?.statusMessage !== conversation.statusMessage
+				|| prevStatus?.statusIcon !== conversation.statusIcon
+				|| prevStatus?.statusClearAt !== conversation.statusClearAt) {
+				emit('user_status:status.updated', {
```

Comment:
Note:
* LeftSidebar gets status data from `this.$store.getters.conversationsList`
* `user_status:status.updated` change participant's status only in RightSidebar for current room, if the participant is this room's attendee

Do you want to soft update participants list also, or only statuses? because in the mixin we purge participants list completely, and add it back from response, then update all participants with the same statuses, which we've just added.

Reply:
> `user_status:status.updated` change participant's status only in RightSidebar for current room, if the participant is this room's attendee

That's not entirely true. It's a global event from the vue library and will affect very place that renders a user status and listens to it, including all avatars (also your own in the top right).

Reply:
Indeed, code in `apps/user_status` is subscribed to it, my bad.

Then a check for changes should be valid and helpful in both places, but there's still a pending question with syncing statuses. We could try to solve it as a follow-up